angular.json

```diff
@@ -14,13 +14,6 @@
             "tsConfig": "tsconfig.lib.json"
           }
         },
-        "test": {
```

Comment:
Why did you remove the test configuration?

Reply:
Because `@angular-builders/jest` is not supported with Jest 26 and correctly Angular CLI 9, 
so you can just call the command:

```
$ jest --config jest.config.js 
```

Reply:
good catch